sql-plugin/src/main/340+/scala/com/nvidia/spark/rapids/shims/GpuSparkPathShims.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims
+
+import com.nvidia.spark.rapids.GpuSparkPath
+
+import org.apache.spark.paths.SparkPath
+
+object GpuSparkPathShims {
+  def to(path: GpuSparkPath) = SparkPath(path.toString)
```

Comment:
Nit: Naming the method `to` makes the code hard to read.  To...what?

IMHO this should be called `toSparkPath`.  On older Spark shims where SparkPath isn't a thing, it returns a String, because that's what a path was in older versions of Spark.  On newer shims, it returns a SparkPath.

Reply:
Thank you for figuring this. We have to keep the names same for all shims. So it was difficult to give it a clear name.
`toSparkPath` is good enough, I think. 👍